### PR TITLE
转换规则 No. 126/128

### PR DESCRIPTION
**Pull request by co63oc**
### PR Docs

https://github.com/PaddlePaddle/PaConvert/issues/112

### PR APIs

```bash
126 torch.cuda.get_rng_state_all
128 torch.cuda.set_rng_state_all
```

### CODE REVIEW

File: tests/test_cuda_get_rng_state_all.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+obj = APIBase("torch.cuda.get_rng_state_all")
+
+
+def test_case_1():
+    pytorch_code = textwrap.dedent(
+        """
+        import torch
+        torch.cuda.get_rng_state_all()
+        result = None
```
Comment (zhwesky2010) [inline tests/test_cuda_get_rng_state_all.py:26]: 可以重写下compare函数，对get_rng_state_all输出结果进行判断下严格一些，可以判断是否属于Generator类型，参考paddle.seed

Comment (co63oc) [reply tests/test_cuda_get_rng_state_all.py]: 已修改

File: tests/test_cuda_get_rng_state_all.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import textwrap
+
+import paddle
+from apibase import APIBase
+
+
+class GetRngStateAllAPIBase(APIBase):
```
Comment (zhwesky2010) [inline tests/test_cuda_get_rng_state_all.py:20]: 这个有点奇特，因为返回的是Generator的list，paddle_result应该是一个list，不过为啥没有报错呢

Comment (co63oc) [reply tests/test_cuda_get_rng_state_all.py:20]: 已修改使用 assert验证

**Review by zhwesky2010 — APPROVED** [review]
LGTM
